qpm.json

```diff
@@ -22,6 +22,12 @@
         ]
       }
     },
+    {
+      "id": "modloader",
```

Comment:
This shouldn't be necessary-- is this still valid?

Reply:
Uhh I assumed it needed to link to modloader